source/connect/favorite-connections/import-export-cli/export.txt

```diff
@@ -0,0 +1,124 @@
+.. _compass-export-connections-cli:
+
+========================================
+Export Favorite Connections with the CLI
```

Comment:
Exporting encrypted connections is very similar to exporting connections unencrypted. The only difference being the used of the `--passphrase=<passphrase>` command line argument.
I think it's good if we guide folks towards using a passphrase if possible.
How do you feel about making the default landing page for `Export Favorite Connections with the CLI` instead show the way to export with a passphrase? And then we can add an extra section about not using a passphrase to export the connections as unencrypted. That way we can avoid having an extra page for that as well?